CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Records Lists - Improve performance (Regression introduced on v4.0.0).
```

Comment:
We may have automated test for this.

Reply:
With a million records 🤔. We will have to change the way our tests are made

Reply:
Of course, it's performance test, they should be apart, and be run only on CI weekly.